stix_shifter/stix_translation/stix_translation.py

```diff
@@ -86,7 +88,7 @@ def translate(self, module, translate_type, data_source, data, options={}, recur
                     dialects_used = 0
                     for dialect in dialects:
                         query_translator = entry_point.get_query_translator(dialect)
-                        if language == query_translator.get_language():
+                        if not language or language == query_translator.get_language():
```

Comment:
is there any possibility of language being None?

Reply:
it is possible, check line 74. if module is provided explicitly then the default language value has to be ignored for the sake of the provided module.
for example:
`it is possible, check line 74. if module is provided explicitly then the default language value has to be ignored for the sake of the provided module.`